packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -712,7 +696,8 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     String? parentId,
     int? sourceLine,
     CpuProfileMetaData? profileMetaData,
-    bool copySampleCountsAndTags = true,
+    bool copyExclusiveSampleCount = true,
+    bool copyInclusiveSampleCount = false,
```

Comment:
can both of these be false by default?

Reply:
can we add some descriptive dart doc about these two parameters explaining when they should be true and false? just to try to avoid any confusion or breakages in the future

Reply:
As discussed offline, `copyExclusiveSampleCount` can't be false by default since the bottom-up tree builder in `profiler_utils.dart` expects a `TreeNode` with the default `shallowCopy()` signature.

I'll add some documentation here explaining them.